gradle-plugin-testing/src/test/groovy/com/palantir/gradle/plugintesting/PluginTestingPluginIntegrationSpec.groovy

```diff
@@ -16,25 +16,32 @@
 
 package com.palantir.gradle.plugintesting
 
-
 import static TestDependencyVersions.resolve
 
-import nebula.test.IntegrationSpec
-import nebula.test.functional.ExecutionResult
-
-class PluginTestingPluginIntegrationSpec extends IntegrationSpec {
+class PluginTestingPluginIntegrationSpec extends AbstractTestingPluginSpec {
 
     private static final String DEPRECATION_ERROR_MESSAGE_FROM_NEBULA = 'Deprecation warnings were found (Set the ignoreDeprecations system property during the test to ignore)'
 
     File specUnderTest
 
     def setup() {
-        //TODO(#xxx): once we have a published version of the plugin, apply it to the project and remove this
-        System.setProperty(TestDependencyVersions.TEST_DEPENDENCIES_SYSTEM_PROPERTY, 'org.junit.jupiter:junit-jupiter:5.11.3,com.netflix.nebula:nebula-test:10.6.1, com.palantir.baseline:gradle-baseline-java:6.4.0')
+        writeDependenciesVersionsFile()
 
-        writeHelloWorld('com.testing')
         //language=gradle
         buildFile << """
+            buildscript {
```

Comment:
I made the test project use GCV since that more closely resembles real-world projects.  There was an issue where projects with GCV were resolving versions of dependencies differently than I so this plugin wasn't actually working